interop-tests/src/test/scala/org/apache/pekko/grpc/interop/GrpcInteropSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2018-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package org.apache.pekko.grpc.interop
+
+import io.grpc.testing.integration.TestServiceHandlerFactory
+
+class GrpcInteropIoWithIoSpec extends GrpcInteropTests(Servers.IoGrpc, Clients.IoGrpc)
+class GrpcInteropIoWithAkkaNettyScalaSpec extends GrpcInteropTests(Servers.IoGrpc, Clients.AkkaNetty.Scala)
+class GrpcInteropIoWithAkkaNettyJavaSpec extends GrpcInteropTests(Servers.IoGrpc, Clients.AkkaNetty.Java)
+class GrpcInteropIoWithAkkaHttpScalaSpec extends GrpcInteropTests(Servers.IoGrpc, Clients.AkkaHttp.Scala)
+//class GrpcInteropIoWithAkkaHttpJavaSpec extends GrpcInteropTests(Servers.IoGrpc, Clients.AkkaHttp.Java)
+
+class GrpcInteropPekkoScalaWithIoSpec extends GrpcInteropTests(Servers.Akka.Scala, Clients.IoGrpc)
+class GrpcInteropPekkoScalaWithAkkaNettyScalaSpec extends GrpcInteropTests(Servers.Akka.Scala, Clients.AkkaNetty.Scala)
+class GrpcInteropPekkoScalaWithAkkaNettyJavaSpec extends GrpcInteropTests(Servers.Akka.Scala, Clients.AkkaNetty.Java)
+class GrpcInteropPekkoScalaWithAkkaHttpScalaSpec extends GrpcInteropTests(Servers.Akka.Scala, Clients.AkkaHttp.Scala)
+//class GrpcInteropPekkoScalaWithAkkaHttpJavaSpec extends GrpcInteropTests(Servers.Akka.Scala, Clients.AkkaHttp.Java)
+
+class GrpcInteropPekkoJavaWithIoSpec extends GrpcInteropTests(Servers.Akka.Java, Clients.IoGrpc)
+class GrpcInteropPekkoJavaWithAkkaNettyScalaSpec extends GrpcInteropTests(Servers.Akka.Java, Clients.AkkaNetty.Scala)
+class GrpcInteropPekkoJavaWithAkkaNettyJavaSpec extends GrpcInteropTests(Servers.Akka.Java, Clients.AkkaNetty.Java)
+class GrpcInteropPekkoJavaWithAkkaHttpScalaSpec extends GrpcInteropTests(Servers.Akka.Java, Clients.AkkaHttp.Scala)
+//class GrpcInteropPekkoJavaWithAkkaHttpJavaSpec extends GrpcInteropTests(Servers.Akka.Java, Clients.AkkaHttp.Java)
+
+//--- Aliases
+
+object Servers {
+  val IoGrpc = IoGrpcJavaServerProvider
+  object Akka {
```

Comment:
I just noticed that this file still has clases/types with the `Akka` name in it, since the PR changes class names as well shall change it here as well?

Reply:
I updated this instance - there are still planty of Akka refs in docs, etc. - and it may be better to leave the other instances until another PR to avoid this one becoming too large

Reply:
No worries, PR is approved anyways so its ready to merge when you are happy with it.